test/active_model_serializers/register_jsonapi_renderer_test_isolated.rb

```diff
@@ -111,18 +101,20 @@ def test_jsonapi_parser_registered
     end
 
     def test_jsonapi_renderer_registered
-      expected = {
-        'data' => {
-          'attributes' => {
-            'name' => 'Johnny Rico'
-          },
-          'type' => 'users'
-        }
-      }
+      expected = {'name' => 'Johnny Rico'}
+
       payload = '{"data": {"attributes": {"name": "Johnny Rico"}, "type": "authors"}}'
       headers = { 'CONTENT_TYPE' => 'application/vnd.api+json' }
       post '/render_with_jsonapi_renderer', params: payload, headers: headers
-      assert expected, response.body
+      assert_equal expected, JSON.parse(response.body)["data"]["attributes"] 
```

Comment:
That was because actual response has tons of other things added to it like meta, relationships which was not part of this test and would break anytime a new element is added.